### DISCUSSION TITLE
[Unknown Mod][Castles] "Town Population Information and Statistics" screen (Well screen), allow the creature information window to be displayed by right-clicking on the creature areas directly

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4800] [Ideas]
Allow the creature information window to be displayed by right-clicking on the creature areas directly in the "Town Population Information and Statistics" screen (Well screen) whether the creature's dwelling is built or not.
This would allow the player to view information about the creature's abilities directly, and this even if the creature's dwelling is not yet built.

In fheroes2:

![Info creatures well 001](https://user-images.githubusercontent.com/43583994/146643183-1df08271-c752-4943-bfb8-468a3996cc67.png)

![Info creatures well 002](https://user-images.githubusercontent.com/43583994/146643189-a17e2c7b-d465-42ce-bc8d-a3fdcd8639a6.png)

![Info creatures well 003](https://user-images.githubusercontent.com/43583994/146643193-98ff9773-7c09-4188-8336-3e98e613a30e.png)

This is not an option.
This can be implemented directly in fheroes2.

**Comment by ihhub** [comment]
I agree to allow this only for already built dwellings. We shouldn't display such info fir non-existing buildings.

**Reply by LeHerosInconnu** [reply]
Hello @ihhub,

> I agree to allow this only for already built dwellings. We shouldn't display such info fir non-existing buildings.

Why shouldn't this be possible?

The player may want to know the creatures' abilities before making a decision to build a particular dwelling.

The player may already control another castle of the same type with the dwelling already built, and want to check the creature's information.
Why force the player to go to the other castle to get the information.
